inits.json

```diff
@@ -45,7 +45,7 @@
             "Align Images? (y/n)": "y",
 
             "Target Star X & Y Pixel": [424, 286],
-            "Comparison Star(s) X & Y Pixel": [[465, 183], [512, 263]]
+            "Comparison Star(s) X & Y Pixel": [[465, 183], [512, 263], [], [], [], [], [], [], [], []]
```

Comment:
What do you think about adding more lists for users to fill in additional comp stars in the `inits.json`? Is it too cluttering? The reason why there's ten in total is because EXOTIC has that limit of 10 comp stars.

Reply:
@tamimfatahi - technically, EXOTIC doesn't have an upper limit on the number of comp stars (I believe that's still the case)

Let's add this into the code for now and see what happens with our users

Reply:
When I added in the `inputs.py` file, I added that upper limit just in case. I can remove it if necessary. It's on line 429 if you want to take a gander!